requirements/base.txt

```diff
@@ -15,6 +15,7 @@ osmnx < 2
 pandas >= 1.5, < 2.1
 polyline >= 2, < 3
 pre-commit < 4
+pyarrow > 14
```

Comment:
Just to ensure that pyarrow can work with GDAL < 3.6 (something windows users might want to force, see: https://github.com/OSGeo/gdal/issues/8959) I would set the minimum to v12. Then cap it to v15, so we don't get caught out by breaking changes that could come in unexpectedly